API/swagger.json

```diff
@@ -269,13 +274,14 @@
             "bearer": []
           }
         ],
+        "description": " Special Terms | Meaning                                     |\n-------------  | --------------------------------------------|\n `*`           | matches any sequence of non-path-separators |\n `.**.`        | matches zero or more directories            |\n `.**{m,M}.`   | matches from m to M directories             |\n\nA doublestar (`**`) should appear surrounded by id separators such as `.**.`.\nA mid-pattern doublestar (`**`) behaves like star: a pattern\nsuch as `path.to.**` would return the same results as `path.to.*`. To apply recursion, the\nid you're looking for is `path.to.**.*`.\nExamples:\nid=path.to.a* will delete all the children of path.to which name starts with a.\nid=path.to.**.a* will delete all the descendant hierarchy of path.to which name starts with a.\nid=path.to.**{1,3}.a* will delete all the grandchildren to great-great-grandchildren of path.to which name starts with a.",
```

Comment:
This is great! But just the classic ** being read as markdown for the examples:
<img width="729" alt="Screenshot 2023-11-30 at 18 35 53" src="https://github.com/ditrit/OGrEE-Core/assets/37706737/2df6f91b-0061-41f0-a7c8-b4ee4300ca5a">